content/articles/domains-xxx.markdown

```diff
@@ -0,0 +1,66 @@
+---
+title: .XXX Domains
+excerpt: This article explains the requirements and special procedures for .XXX domain names.
+categories:
+- Domains
+---
+
+# .XXX Domain Names
+
+* TOC
+{:toc}
+
+---
+
+This article explains the requirements and special procedures for .XXX domain names.
+
+
+## Registration
+In order for a .XXX domain to be activated by the .XXX registry and be allowed to resolve, you must register on [the ICM Registry](http://icmregistry.com/about/sponsored-community/) and receive the required .XXX Member Authorization Token to activate your domain. The .XXX TLD requires a Member Auth Token to be provided in order to activate the domain.
+
+You can provide the token during the registration as an extended attribute:
+![](/files/xxx-member-auth-token.png)
+
+### Registration via the API
```

Comment:
I am a bit confused about this section. Is this expected to show an example of an XXX registration via API? If that's the case, that request is wrong as that is the request to show the extended attributes for XXX, not register one.

I would probably remove it and instead simply extend this sentence:

> You can provide the token during the registration as an extended attribute:

to something like

> You can provide the token during the registration in the UI or as an extended attribute via API:

We should perhaps integrate a bit of explanation of what `xxx_resolving` is (see the link I provided you).

Reply:
I've pushed a new commit - let me know if you'd like any changes!